Client/src/main/java/org/gusdb/oauth2/client/veupathdb/UserProperty.java

```diff
@@ -34,18 +44,13 @@ public void setValue(User user, String value) {
     _setter.accept(user, value);
   }
 
-  public void setName(String name) { _name = name; }
-  public void setDisplayName(String displayName) { _displayName = displayName; }
-  public void setDbKey(String dbKey) { _dbKey = dbKey; }
-  public void setRequired(boolean isRequired) { _isRequired = isRequired; }
-  public void setMultiLine(boolean isMultiLine) { _isMultiLine = isMultiLine; }
-  public void setPublic(boolean isPublic) { _isPublic = isPublic; }
-
   public String getName() { return _name; }
   public String getDisplayName() { return _displayName; }
+  public String getHelpText() { return _helpText; }
+  public String getSuggest() { return _suggest; }
   public String getDbKey() { return _dbKey; }
   public boolean isRequired() { return _isRequired; }
-  public boolean isMultiLine() { return _isMultiLine; }
```

Comment:
Not sure if it matters because i don't know if these methods are used outside of this library, but as these are public methods on a public class, their removal, and the changes to the constructor are all breaking API changes.

Reply:
Ah, this is the server, not a library, nevermind :woman_facepalming:

Reply:
nevermind the nevermind, the repo is named server but contains the client library?

Reply:
Yes, I made the server lib dependent on the client lib to avoid an additional "shared" dependency in "client land".  Yes, realize this is a breaking change; when I merge to master I may release a 2.0 which is a little overdue after the bearer token changes anyway.